[Node.js] - develop branch - module not found @iota/wallet

## Bug description

When updating the instructions on how to get started with node.js for the `develop` branch, I keep getting `Error: Cannot find module '@iota/wallet'` when trying to run any example.

## Rust version

- Rust version: rustc 1.67.1 (d5a82bbd2 2023-02-07)

## Version

`develop`

## Hardware specification

- Operating system: Ubuntu 22.04
- RAM: 16
- Cores: 12
- Device: HP HP Pavilion Gaming Laptop 16-a0xxx

## Steps To reproduce the bug

1. `git clone git@github.com:iotaledger/wallet.rs.git`
2. `cd wallet.rs/bindings/nodejs/`
3. `npm i`
4. `npm run install`
5. `node examples/1-create-account.js`

or simply : ` git clone git@github.com:iotaledger/wallet.rs.git && cd wallet.rs/bindings/nodejs/
 && npm i && npm run install && node examples/1-create-account.js `

## Expected behaviour

The example should run,  and create an account.

## Actual behaviour

The example throws a `MODULE_NOT_FOUND` error  for @iota/wallet

## Errors

```plaintext
node:internal/modules/cjs/loader:958
  throw err;
  ^

Error: Cannot find module '@iota/wallet'
Require stack:
- /home/lucas/Documents/IOTA/tests/buildWalletNode/wallet.rs/bindings/nodejs/examples/1-create-account.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:955:15)
    at Module._load (node:internal/modules/cjs/loader:803:27)
    at Module.require (node:internal/modules/cjs/loader:1021:19)
    at require (node:internal/modules/cjs/helpers:103:18)
    at Object.<anonymous> (/home/lucas/Documents/IOTA/tests/buildWalletNode/wallet.rs/bindings/nodejs/examples/1-create-account.js:6:38)
    at Module._compile (node:internal/modules/cjs/loader:1119:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1173:10)
    at Module.load (node:internal/modules/cjs/loader:997:32)
    at Module._load (node:internal/modules/cjs/loader:838:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/lucas/Documents/IOTA/tests/buildWalletNode/wallet.rs/bindings/nodejs/examples/1-create-account.js'
  ]
}

Node.js v18.8.0
```

--- a/wallet/bindings/nodejs/package.json
+++ b/wallet/bindings/nodejs/package.json
@@ -1,64 +1,64 @@
 {
-  "name": "@iota/wallet",
-  "version": "2.0.3-rc.23",
-  "description": "Node.js binding to the wallet library",
-  "main": "out/lib/index.js",
-  "types": "out/lib/index.d.ts",
-  "scripts": {
-    "lint": "eslint --ignore-path .eslintignore --ext .js,.ts .",
-    "format": "prettier --ignore-path .eslintignore -w {,*/**/}*.{ts,js,json}",
-    "format-check": "prettier --ignore-path .eslintignore -c {,*/**/}*.{ts,js,json}",
-    "build": "node scripts/neon-build && tsc",
-    "build:neon": "cargo-cp-artifact -nc ./index.node -- cargo build --release --message-format=json-render-diagnostics",
-    "docs-wiki-build": "typedoc --githubPages false --disableSources --excludePrivate --excludeInternal --excludeNotDocumented --plugin typedoc-plugin-markdown --theme markdown --hideBreadcrumbs --entryDocument api_ref.md --readme none --hideGenerator --sort source-order --exclude ./**/src/index.ts --out ../../../documentation/docs/references/nodejs ./lib/index.ts ",
-    "prebuild": "prebuild --runtime napi --target 6 --prepack scripts/neon-build.js --strip",
-    "rebuild": "node scripts/neon-build && tsc && node scripts/strip.js",
-    "install": "prebuild-install --runtime napi --tag-prefix='nodejs-binding-v' || npm run rebuild",
-    "test": "jest --forceExit"
-  },
-  "author": "IOTA Foundation <contact@iota.org>",
-  "license": "Apache-2.0",
-  "dependencies": {
-    "@iota/types": "^1.0.0-beta.15",
-    "cargo-cp-artifact": "^0.1.6",
-    "prebuild-install": "^7.1.1",
-    "typescript": "^4.9.4"
-  },
-  "devDependencies": {
-    "@types/jest": "^29.4.0",
-    "@typescript-eslint/eslint-plugin": "^5.30.7",
-    "@typescript-eslint/parser": "^5.30.7",
-    "dotenv": "^16.0.1",
-    "electron-build-env": "^0.2.0",
-    "eslint": "^8.20.0",
-    "eslint-config-prettier": "^8.5.0",
-    "jest": "^29.4.2",
-    "prebuild": "^11.0.4",
-    "prettier": "^2.8.3",
-    "ts-jest": "^29.0.5",
-    "typedoc": "^0.23.24",
-    "typedoc-plugin-markdown": "^3.14.0"
-  },
-  "overrides": {
-    "tar@<=4.4.17": "^4.4.19",
-    "tar@2.0.0": "^4.4.19",
-    "simple-get@1.4.2": "^2.8.2"
-  },
-  "resolutions": {
-    "tar": "^4.4.19",
-    "simple-get": "^2.8.2"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/iotaledger/wallet.rs.git"
-  },
-  "binary": {
-    "napi_versions": [
-      6
-    ]
-  },
-  "bugs": {
-    "url": "https://github.com/iotaledger/wallet.rs/issues"
-  },
-  "homepage": "https://github.com/iotaledger/wallet.rs#readme"
+    "name": "@iota/wallet",
+    "version": "2.0.3-rc.23",
+    "description": "Node.js binding to the wallet library",
+    "main": "out/lib/index.js",
+    "types": "out/lib/index.d.ts",
+    "scripts": {
+        "lint": "eslint --ignore-path .eslintignore --ext .js,.ts .",
+        "format": "prettier --ignore-path .eslintignore -w {,*/**/}*.{ts,js,json}",
+        "format-check": "prettier --ignore-path .eslintignore -c {,*/**/}*.{ts,js,json}",
+        "build": "node scripts/neon-build && tsc",
+        "build:neon": "cargo-cp-artifact -nc ./index.node -- cargo build --release --message-format=json-render-diagnostics",
+        "docs-wiki-build": "typedoc --githubPages false --disableSources --excludePrivate --excludeInternal --excludeNotDocumented --plugin typedoc-plugin-markdown --theme markdown --hideBreadcrumbs --entryDocument api_ref.md --readme none --hideGenerator --sort source-order --exclude ./**/src/index.ts --out ../../../documentation/docs/references/nodejs ./lib/index.ts ",
+        "prebuild": "prebuild --runtime napi --target 6 --prepack scripts/neon-build.js --strip",
+        "rebuild": "node scripts/neon-build && tsc && node scripts/strip.js",
+        "install": "prebuild-install --runtime napi --tag-prefix='nodejs-binding-v' && tsc || npm run rebuild",
+        "test": "jest --forceExit"
+    },
+    "author": "IOTA Foundation <contact@iota.org>",
+    "license": "Apache-2.0",
+    "dependencies": {
+        "@iota/types": "^1.0.0-beta.15",
+        "cargo-cp-artifact": "^0.1.6",
+        "prebuild-install": "^7.1.1",
+        "typescript": "^4.9.4"
+    },
+    "devDependencies": {
+        "@types/jest": "^29.4.0",
+        "@typescript-eslint/eslint-plugin": "^5.30.7",
+        "@typescript-eslint/parser": "^5.30.7",
+        "dotenv": "^16.0.1",
+        "electron-build-env": "^0.2.0",
+        "eslint": "^8.20.0",
+        "eslint-config-prettier": "^8.5.0",
+        "jest": "^29.4.2",
+        "prebuild": "^11.0.4",
+        "prettier": "^2.8.3",
+        "ts-jest": "^29.0.5",
+        "typedoc": "^0.23.24",
+        "typedoc-plugin-markdown": "^3.14.0"
+    },
+    "overrides": {
+        "tar@<=4.4.17": "^4.4.19",
+        "tar@2.0.0": "^4.4.19",
+        "simple-get@1.4.2": "^2.8.2"
+    },
+    "resolutions": {
+        "tar": "^4.4.19",
+        "simple-get": "^2.8.2"
+    },
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/iotaledger/wallet.rs.git"
+    },
+    "binary": {
+        "napi_versions": [
+            6
+        ]
+    },
+    "bugs": {
+        "url": "https://github.com/iotaledger/wallet.rs/issues"
+    },
+    "homepage": "https://github.com/iotaledger/wallet.rs#readme"
 }

